sats-dna/src/main/kotlin/com/sats/dna/components/button/SatsButtonColor.kt

```diff
@@ -22,13 +22,26 @@ internal fun SatsButtonColor.asMaterialButtonColors(isEnabled: Boolean): ButtonC
     )
 }
 
+internal val SatsButtonColor.borderColor: Color?
+    @Composable get() = when (this) {
+        SatsButtonColor.WaitingListSecondary -> SatsTheme.colors.waitingList.primary
+        else -> null
+    }
+
+internal val SatsButtonColor.disabledBorderColor: Color?
+    @Composable get() = when (this) {
+        SatsButtonColor.WaitingListSecondary -> SatsTheme.colors.waitingList.primary
+        else -> null
+    }
+
 internal val SatsButtonColor.backgroundColor: Color
     @Composable get() = when (this) {
         SatsButtonColor.Primary -> SatsTheme.colors.primary.default
         SatsButtonColor.Cta -> SatsTheme.colors.cta.default
         SatsButtonColor.Secondary -> SatsTheme.colors.secondary.default
```

Comment:
There will be updates coming to the secondary button as well. Just need to sort out whether the colors in Figma are correct. And then I'll be adding that change too.